### ISSUE TITLE
Exception/unable to spawn oliphant

### ISSUE THREAD

**Issue by LemADEC** [rwtema/Dark-Menagerie#5]
When trying to use an Elephant with a spawn egg on our server, we get an exception on console:

```
[00:13:05 WARN]: java.lang.reflect.InvocationTargetException
[00:13:05 WARN]:        at sun.reflect.GeneratedConstructorAccessor185.newInstance(Unknown Source)
[00:13:05 WARN]:        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[00:13:05 WARN]:        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[00:13:05 WARN]:        at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.createEntityByID(ItemSpawnEggMenagerie.java:171)
[00:13:05 WARN]:        at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.spawnCreature(ItemSpawnEggMenagerie.java:147)
[00:13:05 WARN]:        at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.func_77648_a(ItemSpawnEggMenagerie.java:78)
[00:13:05 WARN]:        at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:483)
[00:13:05 WARN]:        at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:149)
[00:13:05 WARN]:        at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:489)
[00:13:05 WARN]:        at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:875)
[00:13:05 WARN]:        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
[00:13:05 WARN]:        at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
[00:13:05 WARN]:        at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:244)
[00:13:05 WARN]:        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:173)
[00:13:05 WARN]:        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:980)
[00:13:05 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:423)
[00:13:05 WARN]:        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:798)
[00:13:05 WARN]:        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:658)
[00:13:05 WARN]:        at java.lang.Thread.run(Thread.java:745)
[00:13:05 WARN]: Caused by: java.lang.NoSuchFieldError: base
[00:13:05 WARN]:        at com.rwtema.menagerie.entities.Ents.EntityOliphant.<init>(EntityOliphant.java:240)
[00:13:05 WARN]:        ... 19 more
```

Using dark_menagerie-1.7.10-beta-2.0.jar and cauldron-1.7.10-1.1236.05.212-server.jar

**Comment by LemADEC** [comment]
Other spawn eggs are working fine.

**Comment by therealericbj** [comment]
I am also having this problem.  Except I am not using cauldron.  I have encountered this error using dark_menagerie-1.7.10-beta-2.0 and forge versions 1230, 1232 and 1272 on 2 different servers.

When I attempt to use a spawn egg, I get this message in server console...

[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]: java.lang.reflect.InvocationTargetException
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at java.lang.reflect.Constructor.newInstance(Unknown Source)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.createEntityByID(ItemSpawnEggMenagerie.java:171)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.spawnCreature(ItemSpawnEggMenagerie.java:147)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at com.rwtema.menagerie.items.ItemSpawnEggMenagerie.func_77648_a(ItemSpawnEggMenagerie.java:78)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraftforge.common.ForgeHooks.onPlaceItemIntoWorld(ForgeHooks.java:455)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.item.ItemStack.func_77943_a(ItemStack.java:128)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:389)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]: Caused by: java.lang.NoSuchFieldError: base
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     at com.rwtema.menagerie.entities.Ents.EntityOliphant.<init>(EntityOliphant.java:240)
[06:20:10] [Server thread/INFO] [STDERR/]: [java.lang.Throwable$WrappedPrintStream:println:-1]:     ... 20 more

**Comment by Yucca86** [comment]
This causes Oliphants not to spawn on servers. I have had no Oliphants in my server world, only Oliphants I have seen has been in my singleplayer test world.

**Comment by LemADEC** [comment]
Issue is fixed as of dark_menagerie-3.1a
